SteamKit2/SteamKit2/Util/Utils.cs

```diff
@@ -397,6 +397,12 @@ public static bool TryParseIPEndPoint(string stringValue, out IPEndPoint endPoin
                 return false;
             }
 
+            if (port < ushort.MinValue || port > ushort.MaxValue)

```

Comment:
Could use unit tests for this new codepath - if we have any at all, cause I have a feeling this util isn't tested.

Reply:
The new tests in this PR test the greater than case. I’ll add one for the less than case.

Reply:
May as well test the rest too in case there's no coverage here at all.

Reply:
It’s indirectly covered. 
https://codecov.io/gh/SteamRE/SteamKit/pull/578/src/SteamKit2/SteamKit2/Util/Utils.cs